### PR TITLE
Update pl.ts

### PR DESCRIPTION
**Pull request by ciesiolka**
Align day names as they are printed in official liturgical books

### CODE REVIEW

File: lib/locales/pl.ts [hunk 1]
```diff
@@ -40,8 +40,8 @@ export const locale: Locale = {
 
     easter_time: {
       season: 'Wielkanoc',
-      weekday: '$t(weekdays:{{dow}}, capitalize) {{week}}. tygodnia wielkanocnego',
-      sunday: '{{week}}. Niedziela Wielkanocna',
+      weekday: '$t(weekdays:{{dow}}, capitalize) {{week}} tygodnia wielkanocy',
```
Comment (tukusejssirs) [inline lib/locales/pl.ts]: Isn’t this a typo? Should it not be `Wielkanocy`? 🤔

File: lib/locales/pl.ts [hunk 2]
```diff
@@ -230,7 +230,7 @@ export const locale: Locale = {
       'Św. Dionizego, biskupa i męczennika, patrona miasta i diecezji Saint-Denis',
     divine_mercy_sunday: '2 Niedziela Wielkanocna czyli Niedziela Miłosierdzia Bożego',
     dominic_de_guzman_priest: 'Św. Dominika, prezbitera',
-    easter_sunday: 'Niedziela Wielkanocna Zmartwychwstania Pańskiego',
+    easter_sunday: 'Niedziela Wielkanocy Zmartwychwstania Pańskiego',
```
Comment (tukusejssirs) [inline lib/locales/pl.ts:233]: This sounds strange. I presume `Wielkanocy` should be omitted. Could you verify this please?

**Review by tukusejssirs — CHANGES_REQUESTED** [review]
Thanks, @ciesiolka!

A side note: Would it be possible for you to get a copy of the Polish translation of the Roman Missal (ideally, its latest edition)? I’d be interested esp the GRC celebrations list (which is usually printed on pages around p100), along with the front cover, title page and title page verso. :pray:

**Comment by ciesiolka** [comment]
> Thanks, @ciesiolka!
> 
> A side note: Would it be possible for you to get a copy of the Polish translation of the Roman Missal (ideally, its latest edition)? I’d be interested esp the GRC celebrations list (which is usually printed on pages around p100), along with the front cover, title page and title page verso. 🙏

Sure, but the one printed in polish Missal is from 2009. The most recent one is from 2021 and can be found here: https://liturgia.episkopat.pl/file_get.php?fid=1227

Once I get the RomCal working in my project, I'll update the polish calendar and maybe create sub-calendars for particular dioceses.

Also, I have missals in some "rare" languages. If you want I can provide you the following GRCs:
- Basque
- Croatian
- Greek
- Irish (Gaelic)
- Korean
- Romanian
- Russian
- Ukrainian

File: lib/locales/pl.ts (hunk 2 above)
Comment (ciesiolka) [reply lib/locales/pl.ts:233]: The full, official name (as printed in the 2nd edition of the lectionary released in 2016, which includes linguistic changes that will be followed in the upcoming 3rd ed. of the Roman Missal) is `Niedziela Wielkanocy Zmartwychwstania Pańskiego`.

However, shorter version, used in the lectionary header (next to a page number) is `Niedziela Zmartwychwstania Pańskiego`.

The latter is commonly used, so I'm ok with using the non-official, but familiar-sounding name.

File: lib/locales/pl.ts (hunk 1 above)
Comment (ciesiolka) [reply lib/locales/pl.ts]: Correct, it's a typo. But actually the full name, as printed in the lectionary, would be `tygodnia okresu Wielkanocy`.

**Comment by tukusejssirs** [comment]
Wow! What a collection you have! :eyes: I’d like to have them all! :pray:

Actually, if possible, ideally I’d like to have some pictures like [these](https://drive.google.com/drive/u/3/folders/1F4UwlJ-ajcd_VfeBBiGgMJwKD2Kyj_7T) (those are from the Hungarian Missal, ed typ tertia), i.e. besides the front cover, title page, title page verso, the GRC, it would be helpful to have an example of each celebration name from the Proper of Times, as well as the diocesan celebrations (those are usually printed as an appendix) as well as the list of celebrations (listed in the ToC). That said, even the GRC (along with the FC, TP and TPV) is more that I would wish! Thanks in advance for anything you provide me/us! :pray:

File: lib/locales/pl.ts (hunk 1 above)
Comment (tukusejssirs) [reply lib/locales/pl.ts]: Therefore, should it be rather `tygodnia okresu Wielkanocy`? :thinking: It seems like you prefer `tygodnia Wielkanocy` for some reason. AFAIK, the weekday celebration names are not printed in full in the Roman Missal (they are simply titled by weekday name, e.g. `Monday` unless they are some special days). :thinking:

File: lib/locales/pl.ts (hunk 2 above)
Comment (tukusejssirs) [reply lib/locales/pl.ts:233]: Well, I prefer using official names to the common ones, therefore if `Niedziela Wielkanocy Zmartwychwstania Pańskiego` is the official name from the Lectionary based on the future MR (ed typ tertia), I am okay with that. I just wanted to be sure it is so, as to me (a Slovak), it sounds a bit strange; I would say either `Niedziela Wielkanocy` or `Niedziela Zmartwychwstania Pańskiego`, but I can’t speak Polish (just a bit; I mostly understand it).

**Comment by tukusejssirs** [comment]
@ciesiolka, if you feel like the latest commit is correct, tell me, and I’ll approve the PR for merging. Then we’ll wait for @emagnier to merge it. Thanks for your contribution and I am eagerly awaiting your next ones! :pray:

File: lib/locales/pl.ts (hunk 1 above)
Comment (ciesiolka) [reply lib/locales/pl.ts]: >  AFAIK, the weekday celebration names are not printed in full in the Roman Missal (they are simply titled by weekday name, e.g. `Monday` unless they are some special days). 🤔

In polish missal they are printed in full, i.e. `Sobota drugiego tygodnia Wielkanocnego`.

Anyway, let's keep the change I made as a best compromise.

**Comment by ciesiolka** [comment]
> @ciesiolka, if you feel like the latest commit is correct, tell me, and I’ll approve the PR for merging. Then we’ll wait for @emagnier to merge it. Thanks for your contribution and I am eagerly awaiting your next ones! 🙏

I think the current translation is correct. Let's get this approved and continue on extending other calendars.

**Review by tukusejssirs — APPROVED** [review]

**Review by emagnier — APPROVED** [review]
Thank you @ciesiolka, for your great contribution! 🎉

**Comment by emagnier** [comment]
And thanks to your PR, we have just reached the 1000th commit on this repo 😄

**Pull request merged**